typescript/tools/update-baselines.ts

```diff
@@ -114,7 +114,11 @@ async function main() {
 
     console.log(`Updating baseline for ${library}...`);
     console.log(`  - rm -rf "${baselineDir}"...`);
-    await fsp.rm(baselineDir, {recursive: true});
+    try {
+      await fsp.rm(baselineDir, {recursive: true});
+    } catch (err) {
+      console.log('no baseline to delete');
```

Comment:
If we just ignore this, let's ignore silently. Otherwise, this log messages is not very helpful, should've said "Not removing baseline `${baselineDir}` because it does not exist", or something like that.